### DISCUSSION TITLE
Cookies

### DISCUSSION THREAD

**Discussion by AyameNoop** [deniscerri/ytdlnis#628] [Q&A] [answered]
I exported the cookies from my desktop where i installed a browser extension in Firefox and did login-ed using an incognito window

Now I have the cookies txt file on my phone and no permutation of --cookies /path/to/cookies/file.txt work

Someone please just tell me how to get the damn cookies to work so I can start dling from Youtube again

**Comment by zaednasr** [comment, marked as answer]
Copy the text and go to cookies page. Tap top menu. Import from clipboard
